AssetResolver/src/ZibraVDBAssetResolver.cpp

```diff
@@ -0,0 +1,168 @@
+#include "PrecompiledHeader.h"
+
+#include "ZibraVDBAssetResolver.h"
+
+#include "decompression/DecompressionHelper.h"
+#include "utils/Helpers.h"
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+TF_REGISTRY_FUNCTION(TfDebug)
+{
+    TF_DEBUG_ENVIRONMENT_SYMBOL(ZIBRAVDBRESOLVER_RESOLVER, "Print debug output during ZibraVDB path resolution");
+}
+
+AR_DEFINE_RESOLVER(ZibraVDBResolver, ArResolver);
+
+ZibraVDBResolver::ZibraVDBResolver()
+{
+    UT_Exit::addExitCallback([](void*) {
+        if (Zibra::AssetResolver::DecompressionHelper::HasInstance())
+        {
+            Zibra::AssetResolver::DecompressionHelper::GetInstance().Cleanup();
+        }
+    }, nullptr);
+}
+
+std::string ZibraVDBResolver::_CreateIdentifier(const std::string& assetPath, const ArResolvedPath& anchorAssetPath) const
+{
+    auto assetURI = URI(assetPath);
```

Comment:
please don't use `auto` for types that are not hard to express, it makes code less readable
`URI` is litearlly shorter than `auto`
```suggestion
    URI assetURI = URI(assetPath);
```